### DISCUSSION TITLE
Cannot access setup on another port

### DISCUSSION THREAD

**Discussion by exstntlmsnthrp** [Kovah/LinkAce#473] [Support]
Hello.

I see this discussion section hasn't been active for awhile, so hopefully someone sees this. I installed LinkAce on a network server I have setup. I followed the [setup without docker](https://www.linkace.org/docs/v1/setup/setup-without-docker/). Everything went well until Step 5.

I do have several other things setup on the server (each with a unique port number) without any problem. Given that Apache recommends using config files instead of htaccess files, I set one up for my LinkAce setup as follows. As I have several other things installed on the server, I did give the LinkAce its own port number. Obviously I did enable it and I also restarted Apache.

linkace.config
```
<VirtualHost *:61010>
    ServerName my.local.ip.address
    DocumentRoot /var/www/linkace/public
    
    <Directory /var/www/linkace/public>
        AllowOverride None
        Allow from All
    </Directory>
    
    <Directory /var/www/linkace/public>
        <IfModule mod_negotiation.c>
            Options -Multiviews -Indexes
        </IfModule>
    </Directory>
    
    <Directory /var/www/linkace/public>
        <IfModule mod_rewrite.c>
            RewriteEngine On

            # Handle Authorization Header
            RewriteCond %{HTTP:Authorization} .
            RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]

            # Redirect Trailing Slashes If Not A Folder...
            RewriteCond %{REQUEST_FILENAME} !-d
            RewriteCond %{REQUEST_URI} (.+)/$
            RewriteRule ^ %1 [L,R=301]

            # Handle Front Controller...
            RewriteCond %{REQUEST_FILENAME} !-d
            RewriteCond %{REQUEST_FILENAME} !-f
            RewriteRule ^ index.php [L]
        </IfModule>
    </Directory>
</VirtualHost>
```
I also gave www-data ownership of my linkace directory so there aren't any problems. I should also note that my rewrite and negotiation mods have also been enabled.

However, when I enter my.local.ip.address:61010 or my.local.ip.address:61010/public in my browser, it gives me an unable to connect error. When I nmap my.local.ip.address -p 61010, it is unsuccessful; presumably because linkace is not listening there. Can anyone provide any advice to what is going wrong here? Have I missed something? Is something configured incorrectly?

Any help is greatly appreciated. I really look forward to trying this out!

**Comment by exstntlmsnthrp** [comment]
Can anyone help with this?

**Comment by Kovah** [comment]
Unfortunately, I can't. I haven't working with Apache for like 10 years.

**Comment by martadinata666** [comment]
Not really a helpfull from mine, this is my apache conf, as i rely on `.htaccess` my apache conf pretty standard. 
```
<VirtualHost *:443>
    Protocols h2 http/1.11    
    ServerAdmin webmaster@localhost
    DocumentRoot /var/www/html/public

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined

    SSLEngine on
    SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
    SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
    
    <Directory "/var/www/html/">
        AllowOverride All
        Require all granted
    </Directory>

    <FilesMatch "\.php$">
        SetHandler "proxy:fcgi://localhost:9000/"
        SSLOptions +StdEnvVars
    </FilesMatch>
     
    <Files ~ "^.ht">
        Order allow,deny
        Deny from all
    </Files>

</VirtualHost>

```
